Observer/SalesQuoteProductAddAfter.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace Klaviyo\Reclaim\Observer;
+
+use Klaviyo\Reclaim\Helper\Data;
+
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Catalog\Model\CategoryFactory;
+
+class SalesQuoteProductAddAfter implements ObserverInterface
+{
+    /**
+     * Klaviyo Data Helper
+     * @var Data
+     */
+    protected $_dataHelper;
+
+    /**
+     * Magento Category Factory
+     * @var CategoryFactory
+     */
+    protected $_categoryFactory;
+
+    /**
+     * @param Data $dataHelper
+     * @param CategoryFactory $categoryFactory
+     */
+    public function __construct(
+        Data $dataHelper,
+        CategoryFactory $categoryFactory
+    )
+    {
+        $this->_dataHelper = $dataHelper;
+        $this->_categoryFactory = $categoryFactory;
+    }
+
+    public function execute(Observer $observer)
+    {
+        $quote = $observer->getData('items')[0]->getQuote();
+        $addedItems = $observer->getData('items');
+        $childrenIds = [];
+
+        foreach ($addedItems as $item){
+            if ($item->getProductType() == 'bundle') {
+                $children = $item->getChildren();
+                foreach ($children as $child){
+                    array_push($childrenIds, $child->getId());
+                }
+            }
+
+            if (in_array( $item->getId(), $childrenIds )){
```

Comment:
Going to add a condition to account for this. However, when a bundled item is added, there parent item is the first in the array followed by children, what I've seen in testing. 

But still going to account for this condition to avoid surprises on deployment